CHANGES.rst

```diff
@@ -9,6 +9,7 @@ Improvements:
 
 Bugfix:
  - Failed to send a video captured by the native camera. Replace the file scheme "file://" with "file:/" used by some Android devices.
+ - Fix / Room v3 permalinks don't always work #8315
```

Comment:
Use special link because this is a riot issue (also in commit message - force push to update it)